src/docfx/build/xref/InternalXrefMapBuilder.cs

```diff
@@ -172,13 +137,14 @@ private static InternalXrefSpec[] AggregateXrefSpecs(Context context, string uid
                 return specsWithSameUid;
             }
 
-            // multiple uid conflicts without moniker range definition, drop the uid and log an error
+            // multiple uid conflicts without moniker range definition
+            // order by the declairing file, take the first one
             var conflictsWithoutMoniker = specsWithSameUid.Where(item => item.Monikers.Count == 0).ToArray();
             if (conflictsWithoutMoniker.Length > 1)
             {
-                var orderedConflict = conflictsWithoutMoniker.OrderBy(item => item.Href);
+                var orderedConflict = conflictsWithoutMoniker.OrderBy(item => item.DeclairingFile);
```

Comment:
Only referenced duplicated UID will be reported?

Reply:
The `DeclairingFile` means the file where the user defines the UID, not referencing the UID